library.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "printf",
+  "version": "0.6.3",
+  "description": "Standalone printf/sprintf formatted printing function library",
+  "keywords": "printf, sprintf, formatted printing",
```

Comment:
1. Can a two-word phrase be a keyword?
2. I would add a few more; should I just do it after merging, or is there some policy about what keywords can be used?

Reply:
This looks good. Ultimately, it only affects the accessibility of the library if it gets published in the registry, so there is no harm having lots of keywords.